editors/code/src/rust-analyzer-api.ts

```diff
@@ -111,6 +111,7 @@ export interface SsrParams {
     arg: string;
 }
 export const ssr = request<SsrParams, SourceChange>("ssr");
+export const ssr_check_query = request<SsrParams, Option<String>>("check_ssr_query");
```

Comment:
I guess we should follow `camelCase` convention both for the method name and the variable name here.

Reply:
thank, I didn't know about `npm run lint`